.github/workflows/reusable-node-test.yml

```diff
@@ -38,11 +38,30 @@ jobs:
       - name: Type Check
         run: pnpm type-check
 
-      - name: Node Test
+      - name: Setup Node18 For Testing
```

Comment:
Maybe we could use matrix?

Reply:
Using matrix would case building binding multiple times. Seems a waste.